docs/pbc/all/search/202212.0/third-party-integrations/integrate-algolia.md

```diff
@@ -17,13 +17,13 @@ The Algolia app requires the following Spryker modules:
 * `spryker/category: "^5.11.0"`
 * `spryker/category-storage: "^2.5.0"`
 * `spryker/message-broker-aws: "^1.3.1"`
-* `spryker/price-product: "^4.37.0"`
-* `spryker/product: "^6.29.0"`
-* `spryker/product-approval: "^1.1.0"`
+* `spryker/price-product: "^4.40.0"`
+* `spryker/product: "^6.32.0"`
+* `spryker/product-approval: "^1.1.0"` (Optional)
```

Comment:
Can we remove it if it is optional?

Reply:
There is also an explanation of how to setup plugin for this package which is necessary to wire up when using this package. So I'd like to keep this package.